app/src/main/res/values/strings_activity_settings.xml

```diff
@@ -0,0 +1,59 @@
+<resources>
```

Comment:
This whole file should not exist. 
It was generated by Android studio and accidentally added.